src/panels/config/entities/dialog-entity-editor.ts

```diff
@@ -201,12 +203,21 @@ export class DialogEntityEditor extends LitElement {
     if (!this._entry) {
       return;
     }
+
+    const templates = await fetchTemplateEntities(this.hass);
```

Comment:
This is expensive, to fetch, and await, all template entities on every open op the entity editor.

Reply:
Don't we get the platform from the entity registry?